src/adapter.ts

```diff
@@ -1,16 +1,103 @@
 import { MoveCallTransaction, SuiTransactionResponse } from '@mysten/sui.js';
 import { ConnectResponseType, MartianApis, Permission, SignMessageResponseType } from './types';
+import { Wallet, SUI_DEVNET_CHAIN, SUI_TESTNET_CHAIN, IdentifierArray, WalletAccount, ConnectOutput, SuiSignAndExecuteTransactionInput, SuiSignAndExecuteTransactionOutput } from "@mysten/wallet-standard";
+import { WalletVersion, WalletIcon } from '@wallet-standard/base';
+import { ReadonlyWalletAccount } from "@mysten/wallet-standard";
+// import { WalletAdapter } from "@mysten/wallet-adapter-base";
+
+import {
+  ConnectFeature,
+  ConnectMethod,
+  EventsFeature,
+  EventsOnMethod,
+  SuiSignAndExecuteTransactionFeature,
+  SuiSignAndExecuteTransactionMethod
+} from "@mysten/wallet-standard";
 
 declare const window: {
   martian: { sui: MartianApis };
 };
 
-export class MartianWalletAdapter {
+export class MartianWalletAdapter implements Wallet{
+  icon!: WalletIcon;
+  chains: IdentifierArray = [SUI_DEVNET_CHAIN, SUI_TESTNET_CHAIN];
+  accounts!: readonly WalletAccount[];
```

Comment:
make this empty array, it should fix register wallet not able to connect issues

Reply:
Done